docs/source/configuration/backend.md

```diff
@@ -1,9 +1,8 @@
 # Backend configuration
 
 ## plone.volto
-In order to fully support all Volto features, the Plone backend API needs to be prepared
-for Volto. The add'on `plone.volto` does all the heavy lifting for you and is ready to use
-in your own projects. We used it in our Getting Started section.
+<<<<<<< HEAD
```

Comment:
Broken git merge artifact.

Reply:
fixed.  I was still editing that chapter yesterday. ;-)   (see PR label)